src/console_reporter.cc

```diff
@@ -53,11 +53,10 @@ bool ConsoleReporter::ReportContext(const Context& context) {
 
 void ConsoleReporter::PrintHeader(const Run& run) {
   std::string str =
-      FormatString("%-*s %13s %13s %10s\n", static_cast<int>(name_field_width_),
-                   "Benchmark", "Time", "CPU", "Iterations");
-  if(!run.counters.empty()) {
-    str += " UserCounters...";
-  }
+      FormatString("%-*s %13s %13s %10s%s\n", static_cast<int>(name_field_width_),
```

Comment:
The reporter output tests are expecting the regex `Iterations$`, so adding a space in the format string would make those tests fail, or the tests would have to be adjusted. For that reason, I chose using the space in " UserCounters" as a good balance,